app/root.jsx

```diff
@@ -29,6 +29,18 @@ export default function App() {
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width,initial-scale=1" />
         <script defer data-domain="getalby.com" src="https://squirrel.getalby.com/js/plausible.js"></script>
+        <meta name="title" content="Alby — Lightning buzz for your Browser!"/>
```

Comment:
please move the above <script> tag separately. like this it is hidden inbetween the meta tags.